test/infrastructure/docker/Makefile

```diff
@@ -48,6 +48,10 @@ endif
 CONTROLLER_GEN := $(TOOLS_BIN_DIR)/controller-gen
 CONVERSION_GEN := $(TOOLS_BIN_DIR)/conversion-gen
 GOLANGCI_LINT := $(TOOLS_BIN_DIR)/golangci-lint
+KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
+
+$(KUSTOMIZE): # Build kustomize from tools folder.
+	$(ROOT)/hack/ensure-kustomize.sh
```

Comment:
should this be `make kustomize` from the root makefile (so we are defining how to build Kustomize in single place)?

Reply:
Fixed.